developer-manual/development_process.rst

```diff
@@ -264,6 +289,12 @@ Example of a merged PR with squash: ::
 
     NethServer/dev#1133
 
+Draft pull requests
+-------------------
+
+The use of draft pull requests is recommended to share an on-going development.
+Draft pull requests can be used to test possible implementations of features that do not have an issue yet.
+A draft pull request should always have an assignee.
```

Comment:
Why?

```suggestion
```

Reply:
Because I want to know who is working on a draft, otherwise I will fell free to close it :smiling_imp:

Reply:
Isn't the draft author enough (by default)?

Reply:
Usually not, because some users open the PR and then forgot it for months hoping that someone will jump in :)

If nobody is assignee, nobody is working it: just like the issue.

Reply:
Ok now I understand. In this case this is my proposal:

```suggestion
If the draft pull request does not reference an issue it must have an assignee.
```

That should be true also for normal PRs that do not reference any issue :thinking: